movement_controller:
  ros__parameters:
    use_sim_time: True
    frequency: 30
    cmd_topic: /cmd_vel
    reset_errors: False